data/json/npcs/TALK_COMMON_ALLY.json

```diff
@@ -169,6 +169,12 @@
         "topic": "TALK_DONE",
         "effect": "find_mount"
       },
+      {
+        "text": "Drop off all non favorite items / loot!",
```

Comment:
This sounds very gamey, not something you'd say to a person.
How about "Drop off any items you're not using"
Or "Drop off all the loot you're carrying"

Reply:
woudn't you want to make it clear it's about non favorites?

Reply:
i thought about addind a popup stating something like "this will drop all non favorited items, are you sure" but then thought it would be to annoying to have it come up every time.

Reply:
I think my other suggested change (letting them keep their clothes and weapon) combined with auto-favoriting given items solves this.